components/analytics/components/Summary/SummaryCard.js

```diff
@@ -50,7 +50,7 @@ const SummaryCard = ({ percent, title, count, style, showPercent }) => (
 					flexDirection: count > 100 ? 'column' : 'row',
 				}}
 			>
-				{(+count).toLocaleString()}
+				{typeof count === 'string' ? count : (+count).toLocaleString()}
```

Comment:
can't we just do `count.toString()` here?

Reply:
No, we need to format the number for some cases.